lib/fldabnum.gi

```diff
@@ -100,9 +100,8 @@ InstallGlobalFunction( CyclotomicField, function ( arg )
     if subfield = Rationals then
 
       # The standard field is required.  Look whether it is already stored.
-      if IsBound( CYCLOTOMIC_FIELDS[ xtension ] ) then
-        return CYCLOTOMIC_FIELDS[ xtension ];
-      fi;
+      # If not, generate it and return it
+      return CYCLOTOMIC_FIELDS(xtension);
```

Comment:
Huh, doesn't this lead to an infinite loop?

Has this change been tested? Resp. do the existing test cases cover it?

Reply:
I was indeed misreading this code, so *my* question was stupid! Sorry about that, but great that you added a test case anyway ;-), thanks!